peachjam/signals.py

```diff
@@ -39,3 +42,10 @@ def bg_task_finished(sender, **kwargs):
     if transaction:
         # fake an exit from the transaction context
         transaction.__exit__(None, None, None)
+
+
+@receiver(signals.post_save, sender=CoreDocument)
+def update_language(sender, instance, **kwargs):
+    if not kwargs["raw"]:
+        instance.work.languages = [instance.language.iso_639_3]
```

Comment:
What if the work has now has three different documents all in different languages?